File: Object-oriented programming/Homework_1/HotDrink.java
package geekbrains_course.Homework_1;

public class HotDrink {
    private String name;
    private double volume;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public double getVolume() {
        return volume;
    }

    public void setVolume(double volume) {
        this.volume = volume;
    }

    public HotDrink(String name, double volume) {
        this.name = name;
        this.volume = volume;
    }

    public void Drink(){
        System.out.println("You drink " + this.volume+ " ml " + this.name);
    }

}

File: Object-oriented programming/Homework_7/HotDrink.py
class HotDrink():
    def __init__(self, name:str, volume:float) -> None:
        self._name = name
        self._volume = volume
    
    @property
    def name(self):
        return self._name
    
    @property
    def volume(self):
        return self._volume
    
    @name.setter
    def name(self, name):
        self._name = name
    
    @volume.setter
    def volume(self, volume):
        self._volume = volume

    def HotDrink(self, name:str, volume:float):
        self._name = name
        self._volume = volume
    
    def Drink(self):
        print(f"You drink {self._volume} ml {self._name}")


drink = HotDrink("Tea", 45)
  
drink.volume = 19
  
print(drink.volume, drink.name)